src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+// class TodoForm extends React.Component {
+//   constructor(props) {
+//     super(props);
+//   };
+//
+//   render() {
+//     return (
+//       <div>
+//       </div>
+//     );
+//   };
+// };
+
+const TodoForm = props => {
+  return (
+    <form onSubmit={props.subm}>
+    <input type="text" placeholder="Task to add" name="task" value={props.listItem.task} onChange={props.change}/>
+    <button type="submit">Add</button>

```

Comment:
You can use on `onClick` here to listen to the submitHandler passed down as props. It is called *subm* on line 48 in the App component.
`onClick={props.subm}`